hcs-sxc-java-plugins/hcs-sxc-java-plugins-cryptography/src/main/java/com/hedera/hcs/sxc/plugin/cryptography/cryptography/Cryptography.java

```diff
@@ -1,5 +1,6 @@
-package com.hedera.hcs.sxc.cryptography;
-
+package com.hedera.hcs.sxc.plugin.cryptography.cryptography;
```

Comment:
Probably should name plug-in to match cryptography used here so others can be added later. Cryptography is too generic.